### PR TITLE
use const for --containerized mountns path

### PR DESCRIPTION
**Pull request by dixudx**
**What this PR does / why we need it**:

* make frequently used string `"/rootfs/proc/1/ns/mnt"` as constant var;

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
/cc @euank

**Release note**:

```release-note
None
```

### CODE REVIEW

File: pkg/util/io/writer.go [hunk 1]
```diff
@@ -42,6 +42,11 @@ func (writer *StdWriter) WriteFile(filename string, data []byte, perm os.FileMod
 	return ioutil.WriteFile(filename, data, perm)
 }
 
+const (
+	nsenterPath             = "nsenter"
```
Comment (supereagle) [inline pkg/util/io/writer.go]: As these constants are only used in one method, there is no need to make them as constant vars, just local variables are OK.

Comment (euank) [inline pkg/util/io/writer.go]: Renaming it to `nsenterPath` from `cmd` makes it less clear since it isn't, in fact, a path.

**Comment by dixudx** [comment]
@euank Updated. PTAL. Thanks.

**Comment by euank** [comment]
/lgtm

I can't approve, but it's a rote refactor so I'm sure someone will come through soon enough.

**Comment by euank** [comment]
/lgtm cancel

Actually, I disagree with the current `release-note` status of this. There's no need to include this in a release note IMO since it's a trivial refactor.

The commit message could also be clearer. I'll suggest: `pkg/util: use const for --containerized mountns path` or such.

**Comment by dixudx** [comment]
@euank Updated. PTAL.

**Comment by dixudx** [comment]
@brendandburns @thockin Needs your `/approval no-issue`.

**Comment by dixudx** [comment]
@euank @jingxu97 Rebased. PTAL.

**Comment by dixudx** [comment]
@smarterclayton Need your `/approve no-issue`. Thanks.

**Pull request closed without merging**